services/API-service/src/scripts/scripts.service.ts

```diff
@@ -197,24 +216,31 @@ export class ScriptsService {
               dynamicIndicator: unit,
               adminLevel: adminLevel,
               disasterType: disasterType,
-              eventName: this.getEventName(disasterType),
+              eventName: this.getEventName(disasterType, eventNr),
             });
           }
         }
       }
     }
   }
 
-  private getEventName(disasterType: DisasterType): string {
+  private getEventName(
+    disasterType: DisasterType,
+    eventNr: number = 1,
+  ): string {
     if (disasterType === DisasterType.Typhoon) {
-      return 'Mock typhoon';
+      return `Mock typhoon ${eventNr}`;
     } else {
       return null;
     }
   }
 
-  private getTyphoonLeadTime(): string {
-    return LeadTime.hour72;
+  private getTyphoonLeadTime(eventNr: number = 1): string {
+    if (eventNr === 1) {
+      return LeadTime.hour72;
+    } else if (eventNr === 2) {
```

Comment:
Why are we hard coding eventNr values here? what if we have more number of events